process/tasks/ABProcessElementCore.js

```diff
@@ -411,4 +411,44 @@ module.exports = class ABProcessTaskCore extends ABMLClass {
          return this.wantToDoSomething(instance);
       }
    }
+
+   get startElement() {
```

Comment:
I understood if elements have `.category = "start" `, then they are triggers like `ABProcessTriggerCore.js`, `ABProcessTriggerLifecycleCore.js` and `ABProcessTriggerTimerCore.js`.

If they have more triggers, Could you tell me what/how we know them ?

Reply:
Even though in practice we have only had 1 start element on a diagram, technically speaking we can place > 1 start elements on a diagram and have several ways for us to trigger a process.

This method will only return the 1st element found, when there could be more.   If there are more, how do we know which one to get?

We should rethink how we are using the ```startElement()``` function and have it return all ```startElements()``` it can find.

Reply:
So, you are actually searching for them correctly, but you are just only returning the ```[0]``` entry out of all the possible Start Elements.